mslib/msui/msui_mainwindow.py

```diff
@@ -450,6 +451,7 @@ def __init__(self, local_operations_data=None, tutorial_mode=False, *args):
         self.config_editor = None
         self.local_active = True
         self.new_flight_track_counter = 0
+        self.config_for_gui = config_loader()
```

Comment:
with that new var you can replace a lot lines not individual loading a config options, e.g. https://github.com/annapurna-gupta/MSS/blob/feature/save-view-settings/mslib/msui/msui_mainwindow.py#L458